cronex/__init__.py

```diff
@@ -341,6 +348,9 @@ def parse_atom(parse, minmax):
             prefix, suffix = [int(n) for n in subrange.split('-')]
             if prefix < minmax[0] or suffix > minmax[1]:
                 raise ValueError("\"%s\" is not within valid range." % parse)
+        elif string_is_int(subrange):
```

Comment:
Is there any reason `subrange.isdigit()` wouldn't work here?

Reply:
Not that I can think of.  Updated.